client/app.jsx

```diff
@@ -3,12 +3,35 @@ import ReactDOM from 'react-dom';
 import $ from 'jquery';
 import Producer from './components/Producer';
 import ItemName from './components/ItemName';
-import AnsweredQuestions from './components/AnsweredQuestions';
 import NumberOfRatings from './components/NumberOfRatings';
 import Price from './components/Price';
-import InStock from './components/InStock';
 import ItemDetails from './components/ItemDetails';
-import PurchaseColumn from './components/PurchaseColumn'
+import PurchaseColumn from './components/PurchaseColumn';
+
+const handleStarsMouseEnter = () => {
+  setTimeout(() => {
+    document.getElementsByClassName('Star-Modal')[0].style.display = 'block';
+  }, 250);
+};
+
+const handleStarsMouseLeave = () => {
+  setTimeout(() => {
+    document.getElementsByClassName('Star-Modal')[0].style.display = 'none';
+  }, 250000);
+};
+
+const handleStarModalMouseEnter = () => {
+  setTimeout(() => {
+    document.getElementsByClassName('Star-Modal')[0].style.display = 'block';
+  }, 25);
+};
+
+const handleStarModalMouseLeave = () => {
+  setTimeout(() => {
+    document.getElementsByClassName('Star-Modal')[0].style.display = 'none';
+  }, 200);
+};
+
```

Comment:
What is your thought process for moving these outside of your App class component?

Reply:
These functions don't interact with the specific react component. The task they perform is independent. They don't need to be inside the App class so I didn't put them inside the App class. They could have been left in, but it would be less modular.

Reply:
Ok thanks for the explanation. :)